src/content/en/2020/markup.md

```diff
@@ -163,9 +157,9 @@ Around 89% of pages contain at least one HTML comment, while about 46% of them c
 <![endif]-->
 ```
 
-The above is a non-standard HTML conditional comment. While those have proven to be helpful in the past in order to tackle browser differences, they are history for some time as Microsoft [dropped conditional comments](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh801214(v=vs.85)) in Internet Explorer 10.
+The above is a non-standard HTML conditional comment. While those have proven to be helpful in the past in order to tackle browser differences, they are historical for some time as Microsoft [dropped conditional comments](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh801214(v=vs.85)) in Internet Explorer 10.
```

Comment:
Just to flag this—I’m still not a native speaker but this reads odd. Is this really how one would say this? If you’re sure and this is a solid improvement, feel free to keep it—otherwise I suggest to leave this as is.

Reply:
I struggled with this one myself. Neither reads great IMHO. What about this?:

```suggestion
The above is a non-standard HTML conditional comment. While those have proven to be helpful in the past in order to tackle browser differences, they have been consigned to history for some time as Microsoft [dropped conditional comments](https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh801214(v=vs.85)) in Internet Explorer 10.
```

Reply:
LGTM.